src/diagonal.works/b6/api/functions/math.go

```diff
@@ -27,6 +28,10 @@ func divideInt(context *api.Context, a int, b float64) (float64, error) {
 	return float64(a) / b, nil
 }
 
+func toStr(context *api.Context, a int) string {
```

Comment:
Fine for the moment, but probably we want to make a in interface{}, convert it to a Literal (/Value when they're merged), and add a ToString() method on that?

Reply:
yep ! i ll play weekend